lib/Agrammon/OutputFormatter/PDF.pm6

```diff
@@ -176,6 +177,33 @@ sub input-output-as-pdf(
                 :fr('Set de données'),
             ){$language},
         ),
+        submission => %(
+            farm => %(
+                :de('Betriebsnummer'),
+                :en('Farm number'),
+                :fr("Numéro d'entreprise"),
+            ){$language},
```

Comment:
Maybe some day these want to go in a `.po` file (we have a module for that), but fine for now.

Reply:
Yes, that would make sense.